index.bs

```diff
@@ -643,12 +656,34 @@ The {{VRDisplay}} associated with this event.
 {{VRDisplayEventReason}} describing why this event has has been fired.
 
 
-## Window Interface extension ## {#interface-window}
+## VREventHandlers Interface ## {#interface-vreventhandlers}
 
 <pre class="idl">
-partial interface Window {
+[NoInterfaceObject]
+interface VREventHandlers {
   attribute EventHandler onvrdisplayconnect;
   attribute EventHandler onvrdisplaydisconnect;
+  attribute EventHandler onvrdisplaynavigate;
+};
+</pre>
+
+User agents implementing this specification MUST provide the following new DOM events. The corresponding events must be of type {{VRDisplayEvent}} and must fire on the {{vr}} object. Registration for and firing of the events must follow the usual behavior of DOM4 Events.
+
+<dfn event for="VREventHandlers" id="vreventhandlers-vrdisplayconnect-event">onvrdisplayconnect</dfn>
+A user agent MAY dispatch this event type to indicate that a {{VRDisplay}} has been connected.
```

Comment:
Completely agreed, but maybe as a different PR? (Since we're not changing the definition here, just where it fires from)